src/components/parameter-pages/organize-output-by-swtich.tsx

```diff
@@ -0,0 +1,20 @@
+'use client';
+
+import { organizeOutputByPatientState } from '@/state/organize-output-by-patient-state';
+import { Switch } from '@mantine/core';
+import { useRecoilState } from 'recoil';
+
+export default function OrganizeOutputByPatientSwitch() {
+  const [organizeOutputByPatient, setOrganizeOutputByPatient] = useRecoilState(organizeOutputByPatientState);
+
+  return (
+    <Switch
+      label="Add organizeOutputBy=Patient to the query string"
+      onLabel="YES"
+      offLabel="NO"
+      size="lg"
```

Comment:
Small change for single quotes

Reply:
So prettier makes this double quotes if that is okay, not really a way around that unless we want to change settings but I don't think that is entirely necessary?

Reply:
no worries, I wont be losing sleep over single vs double quotes